### DISCUSSION TITLE
Using MultifileElement throws "Cannot read properties of undefined (reading 'source')"

### DISCUSSION THREAD

**Discussion by everydaysoftware** [vueform/vueform#196] [Questions]
### Environment

Vue 2.7
Vueform 1.9.0

### Reproduction

 ` <Vueform>
    <MultifileElement
      name="multifile"
      label="Multi-file"
      :drop="true"
      :upload-temp-endpoint="{
        url: '/temp',
        method: 'POST',
      }"
    />
  </Vueform>`

### Describe the bug

Using the MultifileElement throws the following error

index.mjs:6145 Uncaught (in promise) TypeError: Cannot read properties of undefined (reading 'source')
    at eval (index.mjs:6145:225)
    at Generator.next (<anonymous>)
    at asyncGeneratorStep (index.mjs:133:1094)
    at _next (index.mjs:133:1408)

<img width="467" alt="image" src="https://github.com/vueform/vueform/assets/78683268/4def9b54-7ddd-4b22-8064-635fbcb6c468">

**Comment by adamberecz** [comment]
This seems to be very similary to a post on Discord:
https://discord.com/channels/787237947635793940/1213062260491288676

The solution was to set the cancelToken when creating an axios instance:
```js
const client = axios.create({
  headers: {
    'X-API-KEY': import.meta.env.VITE_API_KEY,
    'Accept-Language': 'fi'
  },
});
client.CancelToken = axios.CancelToken;
```

Converting this to a discussion as it's not an issue with Vueform.

**Comment by adamberecz** [comment]
@everydaysoftware I've double checked this, but with the provided example code and a simple temp file upload action no errors were produced. The issue is likely to be solved by my previous comment if you are using a custom axios instance.
